### PR TITLE
Enable integration tests for Amazon Linux 2

### PR DESCRIPTION
**Pull request by tilne**
Enable existing integration tests to run on Amazon Linux 2 (alinux2).
Any test that runs for alinux should run for alinux2. Any test that only
ran on alinux should now run on alinux2 instead (e.g., the EFS tests).

Signed-off-by: Tim Lane <tilne@amazon.com>

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: tests/integration-tests/tests/storage/test_fsx_lustre.py [hunk 1]
```diff
@@ -54,6 +54,7 @@ def _test_fsx_lustre_correctly_mounted(remote_command_executor, mount_dir, os):
         "centos7": "defaults,_netdev,flock,user_xattr,noatime,noauto,x-systemd.automount,"
         "x-systemd.requires=lnet.service",
         "alinux": "defaults,_netdev,flock,user_xattr,noatime",
+        "alinux2": "defaults,_netdev,flock,user_xattr,noatime",  # TODO: ensure this is correct
```
Comment (ddeidda) [inline tests/integration-tests/tests/storage/test_fsx_lustre.py]: Has this been tested now?

Comment (tilne) [reply tests/integration-tests/tests/storage/test_fsx_lustre.py]: No. I've only run a subset of the integration tests.

Comment (sean-smith) [reply tests/integration-tests/tests/storage/test_fsx_lustre.py]: @tilne Amazon Linux 2 is systemd based so it has the full set of options in `default`. Rebase and you'll see you don't need this line anymore

File: tests/integration-tests/tests/storage/test_fsx_lustre.py [hunk 2]
```diff
@@ -21,9 +21,9 @@
 
 @pytest.mark.regions(["us-east-1"])
 @pytest.mark.instances(["c5.xlarge"])
-@pytest.mark.oss(["centos7", "alinux"])
+@pytest.mark.oss(["centos7", "alinux", "alinux2", "ubuntu1604", "ubuntu1804"])
```
Comment (sean-smith) [inline tests/integration-tests/tests/storage/test_fsx_lustre.py]: Make this "not centos 6"

File: tests/integration-tests/tests/storage/test_fsx_lustre.py [hunk 3]
```diff
@@ -21,9 +21,9 @@
...
 @pytest.mark.schedulers(["sge"])
-@pytest.mark.usefixtures("os", "instance", "scheduler")
+@pytest.mark.usefixtures("instance", "scheduler")
```
Comment (sean-smith) [inline tests/integration-tests/tests/storage/test_fsx_lustre.py]: Why did you remove "os"?

Comment (tilne) [reply tests/integration-tests/tests/storage/test_fsx_lustre.py]: It isn't needed when one of the other OS markers is used. It wasn't hurting anything I just figured I'd remove unneeded markers if I was changing the file anyway.
